src/client/clientmap.cpp

```diff
@@ -1260,6 +1153,12 @@ void ClientMap::updateDrawListShadow(v3f shadow_light_pos, v3f shadow_light_dir,
 	g_profiler->avg("SHADOW MapBlocks loaded [#]", blocks_loaded);
 }
 
+void ClientMap::reportMetrics(u64 save_time_us, u32 saved_blocks, u32 all_blocks)
+{
+	g_profiler->avg("CM::reportMetrics loaded blocks [#]", all_blocks);
+	g_profiler->avg("CM::reportMetrics cleaned blocks [#]", saved_blocks);
```

Comment:
this will always be zero because ClientMap has `maySaveBlocks() == false`.
relevant code: https://github.com/minetest/minetest/blob/cf5add14728f6f00eec0cc8221050ba91e6a9646/src/map.cpp#L404-L410

Reply:
Removed.